### PR TITLE
Feature/initial setup

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,60 @@
...
+* CVR House Number Element - This is filled with the house number from the result.
+* CVR Postal Code Element - This is filled with the postal code from the result.
+* CVR City Element - This is filled with the city from the result.
+
+## Coding standards
+
+Run phpcs with the provided configuration:
+
+```shell
+/path/to/your/phpcs --standard=phpcs.xml.dist
```
Comment (rimi-itk) [inline README.md]: Consider using `"scripts"` in `composer.json` to make it easier to run this, i.e. something like

```
composer install
composer coding-standards-check
```

File: src/Plugin/WebformElement/CvrCityElement.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace Drupal\os2forms_cvr_lookup\Plugin\WebformElement;
+
+use Drupal\webform\Plugin\WebformElement\TextField;
+
+/**
+ * @WebformElement(
+ *   id = "cvr_city_element",
+ *   label = "CVR City Element",
+ *   description = "CVR City Element description",
```
Comment (rimi-itk) [inline src/Plugin/WebformElement/CvrCityElement.php]: This is true, but a very generic description.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: src/Plugin/WebformElement/CvrCityElement.php (hunk 2 above)
Comment (lats1) [reply src/Plugin/WebformElement/CvrCityElement.php]: Fixed in 636b1b1

File: README.md (hunk 1 above)
Comment (lats1) [reply README.md]: Fixed in 
f1a1de9

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**